File: pedro019/teste_fluxogramal/programação_basica/questao11.py
n1 = int(input('digite a primeira nota: '))
n2 = int(input('digite a segunda nota: '))
n3 = int(input('digite a terceira nota: '))
n4 = int(input('digite a quarta nota: '))
p1 = int(input('digite o primeiro peso: '))
p2 = int(input('digite o segundo peso: '))
p3 = int(input('digite o terceiro peso: '))
p4 = int(input('digite o quarto peso: '))

nota_geral = n1 * p1 + n2 * p2 + n3 * p3 + n4 *p4 / p1 + p2 + p3 + p4
print('sua nota ponderada é: ' + str(nota_geral))

File: vinicius012/programacao_basica/Questao11.java
import java.util.Scanner;

public class Questao11{
    public static void main(String[] args) {
    Scanner s;
    s = new Scanner(System.in);

    System.out.println("Digite o valor da primeira nota: ");
    double a = s.nextDouble();
    System.out.println("Digite o valor da segunda nota: ");
    double b = s.nextDouble();
    System.out.println("Digite o valor da terceira nota: ");
    double c = s.nextDouble();
    System.out.println("Digite o valor da quarta nota: ");
    double d = s.nextDouble();

    System.out.println("Digite o valor do primeiro peso: ");
    double e = s.nextDouble();
    System.out.println("Digite o valor do primeiro peso: ");
    double f = s.nextDouble();
    System.out.println("Digite o valor do primeiro peso: ");
    double g = s.nextDouble();
    System.out.println("Digite o valor do primeiro peso: ");
    double h = s.nextDouble();


    double x = a * e + b * f + c * g + d * h / e + f + g + h;

    System.out.println("a media e " + x);
    }
}
